src/types/feeBreakdown.ts

```diff
@@ -1,8 +1,15 @@
 export interface FeeBreakdown {
   amount: {
     calculated: number;
+    calculatedVAT?: number;
     payable: number;
-    vat: number;
+    payableVAT?: number;
+    fastTrack?: number;
+    fastTrackVAT?: number;
+    serviceCharge?: number;
+    serviceChargeVAT?: number;
+    paymentProcessing?: number;
+    paymentProcessingVAT?: number;
```

Comment:
Same as previous comments, these would be just `number` not `number | undefined`.